src/schema/v2/notifications/index.ts

```diff
@@ -56,6 +57,10 @@ export const NotificationType = new GraphQLObjectType<any, ResolverContext>({
       ...date(({ date }) => date),
       deprecationReason: "Please use `publishedAt` instead",
     },
+    content: {
+      type: NotificationContentType,
```

Comment:
notification types should live under the `/notifications/` directory

Reply:
True, the `content`, or whatever naming we come up with - can move to the `notifications/` directory.